cartographer_ros/configuration_files/backpack_3d.lua

```diff
@@ -18,6 +18,14 @@ include "sparse_pose_graph.lua"
 options = {
   sparse_pose_graph = SPARSE_POSE_GRAPH,
   trajectory_builder = TRAJECTORY_BUILDER_3D,
+  map_frame = "map",
+  odom_frame = "odom",
+  tracking_frame = "base_link",
+  provide_odom = true,
+  laser_min_range_m = 0.,
+  laser_max_range_m = 30.,
+  laser_missing_echo_ray_length_m = 5.,
+  num_3d_lasers = 2
```

Comment:
Can we try to settle on consistent placement of 2D/3D? I vote for at the end (e.g. LaserScan2D, LaserScan3D, num_lasers_3d, etc.).